src/test/javascript/spec/component/overview/exercise-details/exercise-details-student-actions.component.spec.ts

```diff
@@ -33,6 +33,8 @@ import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
 import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
 import { MockProfileService } from '../../../helpers/mocks/service/mock-profile.service';
 import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
+import { MockParticipationService } from '../../../helpers/mocks/service/mock-participation.service';
```

Comment:
can you please fix the unnecessary import?
'MockParticipationService' is defined but never used